docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -201,116 +204,116 @@ while (json.Read())
-| `full-date`       | `date-fullyear "-" date-month "-" date-mday`               | 5.2.2.1b             | Extended calendar date                                                                                          |
-| `full-time`       | `partial-time time-offset`                                 | 5.3.3 or 5.3.4.2     | 5.3.3 is "UTC of day" while 5.3.4.2 is "Local time of day with the time shift between local time scale and UTC" |
-| `date-time`       | `full-date "T" full-time`                                  | 5.4.2.1              | Extended calendar date and time of day                                                                          |
+The extended ISO 8601-1:2019 profile implemented in <xref:System.Text.Json> defines the following components for
+date and time representations. These components are used to define various levels of date and time granularity
+supported when parsing and formatting <xref:System.DateTime> and <xref:System.DateTimeOffset> representations.
+
+| Component       | Format                      | Description                                                                     |
+|-----------------|-----------------------------|---------------------------------------------------------------------------------|
+| Year            | "yyyy"                      | 0001-9999                                                                       |
+| Month           | "MM"                        | 01-12                                                                           |
+| Day             | "dd"                        | 01-28, 01-29, 01-30, 01-31 based on month/year                                  |
+| Hour            | "HH"                        | 00-23                                                                           |
+| Minute          | "mm"                        | 00-59                                                                           |
+| Second          | "ss"                        | 00-59                                                                           |
+| Second fraction | "FFFFFFF"                   | Minimum of one digit, maximum of 16 digits                                      |
+| Time offset     | "K"                         | Either "Z" or "('+'/'-')HH':'mm"                                                |
+| Partial time    | "HH':'mm':'ss[FFFFFFF]"     |                                                                                 |
+| Full date       | "yyyy'-'MM'-'dd"            |                                                                                 |
+| Full time       | "'Partial time'K"           | UTC of day or Local time of day with the time offset between local time and UTC |
+| Date time       | "'Full date''T''Full time'" | Extended calendar date and time of day                                          |
```

Comment:
Can you add an example DateTime format? 

Also, I don't see "extended" being used anywhere in https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings / https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings.

What does it mean?

Reply:
Example added, "extended" removed.